app/src/main/java/com/asfoundation/wallet/navigator/TransactionViewNavigator.kt

```diff
@@ -49,6 +50,9 @@ class TransactionViewNavigator(private val settingsRouter: SettingsRouter,
     context.startActivity(intent)
   }
 
+  fun openAptoide(context: Context) {
```

Comment:
Why not use the navigateToBrowser function, by passing the Uri from the model to the navigator?